index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    
+    <main id="main">
+      <h1 id="title">Dr. Norman Borlaug</h1>
+      <div id="img-div">
```

Comment:
try using the section tag instead of the div, this will make your code cleaner and easier to understand

Reply:
- [ ] Please make sure your comment is more polite. So it should also be a suggestion instead of making it look like an order.  For example, you can say, "**It will be better** to use the `section` tag instead of the `div`, this will make your code cleaner and easier to understand, Kindly, work on refactoring your code".  ^_^

- [ ] . What do you think to include some links to make your review as an opportunity for the student to learn -->  [HTML Semantic Elements](https://www.w3schools.com/html/html5_semantic_elements.asp) 🦾